### PR TITLE
Crash fixes

### PR DESCRIPTION
**Pull request by BingShearer**
Fixed a couple bugs which would cause the Interface to crash by reordering some code.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
An admin must verify this patch for the pull request build to fire. Type "ok to test" to begin testing this pull request or "add to whitelist" to test this and all future pull requests from this collaborator.

File: interface/src/ui/overlays/ImageOverlay.cpp [hunk 1]
```diff
@@ -75,46 +75,49 @@ void ImageOverlay::render(RenderArgs* args) {
...
-        DependencyManager::get<GeometryCache>()->renderQuad(topLeft, bottomRight, texCoordTopLeft, texCoordBottomRight, quadColor);
-    } else {
-        DependencyManager::get<GeometryCache>()->renderQuad(topLeft, bottomRight, quadColor);
-    }
-
-    if (_renderImage) {
-        glDisable(GL_TEXTURE_2D);
+        if (_renderImage) {
+            glDisable(GL_TEXTURE_2D);
+        }
```
Comment (AndrewMeadows) [inline interface/src/ui/overlays/ImageOverlay.cpp]: The above **if (_renderImage)** check is now redundant.

You've also lost the render of the quad when _renderImage is false.  That is, there should now two cases when we would render an untextured quad:

(1) _renderImage is false (this case was lost)

(2) the texture has a zero width or height (this case survived)

**Comment by murillodigital** [comment]
add to whitelist

**Comment by hifi-gustavo** [comment]
Build console output for commit: 99c563602f34349614b409e7b96e6663213485d6 available at: https://gist.github.com/356d723e65f42733181b<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5290.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5290.dmg

**Pull request merged**